packages/webview_flutter/example/tizen/Runner.csproj

```diff
@@ -2,7 +2,7 @@
 
   <PropertyGroup>
     <OutputType>Exe</OutputType>
-    <TargetFramework>tizen60</TargetFramework>
+    <TargetFramework>tizen40</TargetFramework>
```

Comment:
Could you also update `tizen-manifest.xml` of this example so that the `api-version="6"` attribute matches the target framework?

Reply:
I've update it. I didn't know that it means .Net API version until you told me!